### ISSUE TITLE
Model implementation

### ISSUE THREAD

**Issue by Juanfra21** [Juanfra21/text2sql#5]
Set Up the Training Environment: Use TensorFlow or PyTorch to set up the model. Define model architecture, loss functions, and optimizers.
